satpy/tests/test_utils.py

```diff
@@ -307,3 +312,26 @@ def test_specific_check_satpy(self):
                     checked_fake = True
             self.assertTrue(checked_fake, "Did not find __fake module "
                                           "mentioned in checks")
+
+
+def test_debug_on(caplog):
+    """Test that debug_on is working as expected."""
+    from satpy.utils import debug_on
+
+    def depwarn():
+        logger = logging.getLogger("satpy.silly")
+        logger.debug("But now it's just got SILLY.")
```

Comment:
A grammar comment: I would have written "it's just gotten SILLY" or "it just got SILLY", but I'm no native speaker...
https://grammarist.com/usage/got-gotten/
Maybe @djhoese and @simonrp84 can comment? (Note that the official language of pytroll is US-english...)

Reply:
https://www.quotes.net/mquote/3511

Reply:
Ah, well, nothing trumps a quote from Monty Python :smile: